lib/auth/touchid/attempt.go

```diff
@@ -64,3 +64,15 @@ func AttemptLogin(origin, user string, assertion *wanlib.CredentialAssertion, pi
 	}
 	return resp, actualUser, nil
 }
+
+// AttemptDeleteNonInteractive attemts deletes a Secure Enclave credentials.
+// Does not require user interaction.
+func AttemptDeleteNonInteractive(credentialID string) error {
+	if !IsAvailable() {
+		return ErrNotAvailable
+	}
+	if credentialID == "" {
+		return errors.New("credentialID required")
+	}
+	return native.DeleteNonInteractive(credentialID)
```

Comment:
WDYT about following AttemptLogin and wrapping ErrNotAvailable/ErrCredentialNotFound in ErrAttemptFailed? It makes the error handling a bit simpler by pulling the knowledge of specific errors here instead.